proto/buf/alpha/registry/v1alpha1/repository.proto

```diff
@@ -244,3 +246,13 @@ message ListRepositoryContributorsResponse {
   // There are no more pages if this is empty.
   string next_page_token = 2;
 }
+
+message GetRepositorySettingsRequest {
+  // The ID of the repository for which to get the settings.
+  string repository_id = 1;
+}
+
+message GetRepositorySettingsResponse {
+  // The number of outside contributors in the repository.
```

Comment:
Does this include implicit members with an explicit role?

Reply:
no, it's not, will add this to the comment